locale/en/auth.ftl

```diff
@@ -0,0 +1,314 @@
+postVerify-title = Next sync between your devices!
+postVerify-description = Sync privately keeps your bookmarks, passwords and other { -brand-firefox } data the same across all your devices.
+postVerify-subject = Account verified. Next, sync another device to finish setup
+postVerify-setup = Set up next device
+postVerify-action = Set up next device
+# Variables:
+#  $email (String) - Link to https://accounts.firefox.com/support
+postVerify-support = Have questions? Visit { $supportUrl }
+
+postVerifySecondary-subject = Secondary email added
+postVerifySecondary-title = Secondary email added
+# Variables:
+#  $secondaryEmail (String) - A user's secondary email address
+postVerifySecondary-description = You have successfully verified { $secondaryEmail } as a secondary email from your { -product-firefox-account }. Security notifications and sign-in confirmations will now be delivered to both email addresses.
+postVerifySecondary-action = Manage account
+
+recovery-subject = Reset your password
+recovery-title = Need to reset your password?
+recovery-description = Click the button within the next hour to create a new password. The request came from the following device:
+recovery-action = Create new password
+
+# Variables:
+#  $productName (String) - The name of the subscribed product, e.g. Mozilla VPN
+subscriptionPaymentExpired-subject = Credit card for { $productName } expiring soon
+subscriptionPaymentExpired-title = Your credit card is about to expire
+# Variables:
+#  $productName (String) - The name of the subscribed product, e.g. Mozilla VPN
+subscriptionPaymentExpired-content = The credit card you’re using to make payments for { $productName } is about to expire.
+
+# Variables:
+#  $productName (String) - The name of the subscribed product, e.g. Mozilla VPN
+subscriptionReactivation-subject = { $productName } subscription reactivated
+# Variables:
+#  $productName (String) - The name of the subscribed product, e.g. Mozilla VPN
+subscriptionReactivation-title = Thank you for reactivating your { $productName } subscription!
+# Variables:
+#  $invoiceTotal (String) - The amount of the subscription invoice, including currency, e.g. $10.00
+#  $nextInvoiceDateOnly (String) - The date of the next invoice, e.g. 2016/01/20
+subscriptionReactivation-content = Your billing cycle and payment will remain the same. Your next charge will be { $invoiceTotal } on { $nextInvoiceDateOnly }. Your subscription will automatically renew each billing period unless you choose to cancel.
+
+subscriptionsPaymentExpired-subject = Credit card for your subscriptions is expiring soon
+subscriptionsPaymentExpired-title = Your credit card is about to expire
+subscriptionsPaymentExpired-content = The credit card you’re using to make payments for the following subscriptions is about to expire.
+# Variables:
+#  $productName (String) - The name of the subscribed product, e.g. Mozilla VPN
+subscriptionsPaymentExpired-name = { $productName }
```

Comment:
Do we actually need this exposed as an FTL message, since it's just the product name?

Reply:
I don't think so, will remove 👍